src/main/scala/com/datastax/gatling/plugin/response/DseResponseHandler.scala

```diff
@@ -26,118 +27,145 @@ import io.gatling.core.stats.message.ResponseTimings
 
 import scala.util.Try
 
+object DseResponseHandler {
+//  private def removeNewLineChars(str: String) = str.replaceAll("""(\r|\n)""", " ")
```

Comment:
Any reason why we are keeping this around?

Reply:
No, removing